python/cuml/test/dask/test_kneighbors_classifier.py

```diff
@@ -158,8 +158,9 @@ def test_predict(dataset, datatype, n_neighbors, n_parts, batch_size, client):
     assert accuracy_score(y_test, distributed_out[0]) > 0.12
 
 
+@pytest.mark.skip(reason="Sometimes incorrect labels are returned")
```

Comment:
This seems like a real problem! @divyegala can you file an issue for this one, please?

Reply:
had an offline chat with Divye. He mentioned that @cjnolet is already working on the fixes. Resolving this and the other comment I made previously.

Reply:
Small correction here- @wxbn is actually working on the fixes. I was just helping to triage the issue so we could get this PR merged.